client/extensions/woocommerce/lib/countries/index.js

```diff
@@ -44,3 +44,21 @@ export const getStateData = ( country, state ) => {
 
 	return stateData;
 };
+
+/**
+ * Return a "sorted" list of countries, with a subset pulled to the top,
+ * and the rest sorted alphabetically.
+ * @param {array} list  List of countries to sort
+ * @return {array} sorted list of countries
+ */
+export function sortPopularCountriesToTop( list ) {
+	const popularCodes = [ 'AU', 'BR', 'CA', 'FR', 'DE', 'IT', 'ES', 'SE', 'GB', 'US' ];
```

Comment:
Out of curiosity - is this list taken from woo core? I expect some people would be sensitive to their countries not being included in lists like this, so always wondered how they are derived.

Reply:
I didn't think to look at WC core, this was actually derived from the country field when editing domain contact info (something I happened to be looking at recently). After a quick look I don't think WC has a list like this.

Reply:
Gotcha - well if it is already in Calypso that way, we can stick with it. I was just more curious about it really.